tap/passive_tapper.go

```diff
@@ -95,6 +97,14 @@ func StartPassiveTapper(opts *TapOpts, outputItems chan *api.OutputChannelItem,
 		tapTargets = opts.FilterAuthorities
 	}
 
+	if *tls {
+		for _, e := range extensions {
+			if e.Protocol.Name == "http" {
```

Comment:
Why is it HTTP only? The other protocols might also be encrypted through `libssl.so`

Reply:
Currently we support https only, i'm not sure how libssl works with other protocols

Reply:
It depends on implementation, right?
What's the harm in passing the data to all extensions anyway?

Reply:
@nimrod-up9 The harm is to introduce a path of code that we don't know if it works, its like gambling. We do know how openssl works with https, I have no idea how it works with other protocols if any..

Reply:
I'll add a task to check.
I'm not sure I agree in this case. That's just how the whole extensions mechanism works. Each extensions takes a stream of bytes and decides for itself if it should tap it or not.